public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -311,9 +311,15 @@ export default class InfluxDatasource extends DataSourceWithBackend<InfluxQuery,
 
     if (query.tags) {
       expandedQuery.tags = query.tags.map((tag) => {
+        let val = tag.value;
```

Comment:
Here's a unit test for this: 

https://github.com/grafana/grafana/pull/70101/files#diff-62cbde0cbe63fcdf60ae080a5f796eda77d9529ba6f9bfc1da9a618e735b047cR161-R179